README.md

```diff
@@ -1 +1,120 @@
-# My-First-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+   <br/>
```

Comment:
- [ ] As a comment/guide, this section should not be committed to your project. This makes your work look professional and have your codebase clean 👍